TopMovies/View/MovieCollectionViewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MovieCollectionViewCell.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 08.02.2021.
+//
+
+import UIKit
+import SnapKit
+import Nuke
+
+class MovieCollectionViewCell: UICollectionViewCell {
+    private let posterImage = UIImageView()
+    private let adultLabel = UILabel()
+    private let ratingLabel = UILabel()
+    private let titleLabel = UILabel()
+    private let descriptionLabel = UILabel()
+    private let shortInfoStackView = UIStackView()
+    private let infoStackView = UIStackView()
+    
+    public func configureWith(movie: MovieCollectionProps) {
+        if let url = movie.posterURL {
+            let options = ImageLoadingOptions(placeholder: movie.posterPlaceholderImage,
+                                              transition: .fadeIn(duration: 0.2))
+            Nuke.loadImage(with: url,
+                           options: options,
+                           into: posterImage)
+        } else { posterImage.image = movie.posterPlaceholderImage }
```

Comment:
That's almost it, but since it takes time for the image to be loaded, you should always set a placeholder before load a new one:
```swift
posterImage.image = movie.posterPlaceholderImage 
let options = ImageLoadingOptions(
  placeholder: movie.posterPlaceholderImage, 
  transition: .fadeIn(duration: 0.2)
)
Nuke.loadImage(
  with: url, 
  options: options, 
  into: posterImage
)
```